FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,51 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, response)):
+
+				completion(FeedItemMapper.map(data, response: response))
+
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+internal final class FeedItemMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+
+		var feed: [FeedImage] {
+			return items.map {
+				$0.item
+			}
+		}
+	}
+
+	private struct Item: Decodable {
+		let image_id: UUID
+		let image_desc: String?
+		let image_loc: String?
+		let image_url: URL
+
+		var item: FeedImage {
+			return FeedImage(id: image_id, description: image_desc, location: image_loc, url: image_url)
+		}
+	}
+
+	static var OK_200: Int { return 200 }
```

Comment:
`OK_200` is an implementation detail of `FeedItemMapper`, and should be marked as `private`:
```suggestion
	private static var OK_200: Int { return 200 }
```